server/package.json

```diff
@@ -25,5 +26,13 @@
   "homepage": "https://github.com/Anirudh94/boreas#readme",
   "dependencies": {
     "express": "^4.14.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-babili": "0.0.9",
+    "babel-preset-es2015": "^6.18.0",
+    "babelify": "^7.3.0",
+    "babili": "0.0.9",
+    "browserify": "^13.1.1"
   }
```

Comment:
Ah gotcha. I recall reading about this before now but I didn't get an implementation. Thanks!